WordPress/gutenberg
# File Block: Makes the button an element button

## What?
Makes the button in the file block a button element.

## Why?
This enables themes to target the button in file blocks without custom CSS.

## How?
Adds the experimental class name to the button. The shortcoming of the approach is that it won't work for blocks that have already been serialised. The alternative approach is to add the file block button class name to the list of selectors in the element selector, but this will cause bloat to this selector which we will need to support forever.

This approach is a compromise which will work for newly added form blocks, but not existing ones.

## Testing Instructions
- Use this PR: https://github.com/Automattic/themes/pull/5849
- Switch to Archeo
- Create a new post
- Add a file block
- Confirm that the button looks the same as other buttons in the editor and the frontend

## Screenshots or screencast 
<img width="652" alt="Screenshot 2022-05-23 at 13 15 23" src="https://user-images.githubusercontent.com/275961/169817133-028b5ce9-1048-45f9-b1f8-e23bf5ea97be.png">

cc @WordPress/block-themers
